### PR TITLE
KFS tensor proto deserialization

### PR DESCRIPTION
**Pull request by mkuczyns11**

### CODE REVIEW

File: src/precision.hpp [hunk 1]
```diff
@@ -107,6 +107,49 @@ inline static Precision fromString(const std::string& s) {
...
+        {"FP32", Precision::FP32},
+        {"FP16", Precision::FP16},
+        {"INT64", Precision::I64},
+        {"INT32", Precision::I32},
+        {"INT16", Precision::I16},
+        {"INT8", Precision::I8},
+        {"UINT64", Precision::U64},
+        {"UINT32", Precision::U32},
+        {"UINT16", Precision::U16},
+        {"UINT8", Precision::U8}};
```
Comment (atobiszei) [inline src/precision.hpp:123]: It's worthy here to add TODO for BYTES datatype
```suggestion
        {"UINT8", Precision::U8}};
        // {"BYTES", Precusion::??} // TODO decide what to do with BYTES 
```

File: src/precision.hpp [hunk 2]
```diff
@@ -107,6 +107,49 @@ inline static Precision fromString(const std::string& s) {
...
+        {Precision::FP16, "FP16"},
+        {Precision::I64, "INT64"},
+        {Precision::I32, "INT32"},
+        {Precision::I16, "INT16"},
+        {Precision::I8, "INT8"},
+        {Precision::U64, "UINT64"},
+        {Precision::U32, "UINT32"},
+        {Precision::U16, "UINT16"},
+        {Precision::U8, "UINT8"},
+        {Precision::BOOL, "BOOL"},
```
Comment (atobiszei) [inline src/precision.hpp:144]: It's worth to list here known currently unsupported datatypes with TODO's

File: src/precision.hpp (hunk 1 above)
Comment (mkuczyns11) [reply src/precision.hpp:124]: done

File: src/precision.hpp (hunk 2 above)
Comment (mkuczyns11) [reply src/precision.hpp:145]: done

**Review by atobiszei — APPROVED** [review]

**Review by ficol — APPROVED** [review]

**Pull request merged**